prometheus/handlers/handlers_test.go

```diff
@@ -71,7 +74,7 @@ const (
 func TestGetConfigureAlertHandler(t *testing.T) {
 	// Successful Post
 	client := &mocks.PrometheusAlertClient{}
-	client.On("ValidateRule", sampleAlert1).Return(nil)
+	//client.On("ValidateRule", sampleAlert1).Return(nil)
```

Comment:
should we delete this line?  Or what's going on here?

Reply:
yeah I'll fix that :)